.changeset/lazy-bats-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-solana-program": patch
```

Comment:
Should this be a major, rather?

Reply:
Let's do minor since we're still under `0.x`.